code/script.js

```diff
@@ -31,17 +34,140 @@ const showMessage = (message, sender) => {
+  <button id="nähenFortgeschrittenBtn">ganz okay</button>
+  <button id="nähenExpertenBtn">ziemlich gut!</button>
+  `
+    document.getElementById('nähenBeginnerBtn').addEventListener('click', handleBeginnerInput);
+    document.getElementById('nähenFortgeschrittenBtn').addEventListener('click', handleFortgeschrittenInput);
+    document.getElementById('nähenExpertenBtn').addEventListener('click', handleProInput);
+
+}
+
+const handleBeginnerInput = () => {
+  showMessage("Beginner", 'user')
+
+  setTimeout(() => showBeginnerMessage(), 1000)
+
+}
+
+const showBeginnerMessage = () => {
+  showMessage("Cool. Ich freue mich sehr dir das Nähen beizubringen. Zusammen werden wir schnell zum Nähprofi!", 'bot')
+
+  setTimeout(() => showKontaktMessage(), 1800)
+}
+
+const handleFortgeschrittenInput = () => {
+  showMessage("Fortgeschritten", 'user')
+
+  setTimeout(() => showFortgeschrittenMessage(), 1000)
+
+}
+
+const showFortgeschrittenMessage = () => {
+  showMessage("Yes! Ich freue mich gemeinsam noch mehr zu lernen", 'bot')
+
+  setTimeout(() => showKontaktMessage(), 1800)
+}
+
+const handleProInput = () => {
+  showMessage("Experte", 'user')
+
+  setTimeout(() => showProMessage(), 1000)
+
+}
+
+const showProMessage = () => {
+  showMessage("Cool, ich freue mich sehr auf herausfordernde Projekte", 'bot')
+
+  setTimeout(() => showKontaktMessage(), 1800)
+}
+
+const showKontaktMessage = () => {
+  showMessage("Wie kann ich dich am besten erreichen?", 'bot')
+
+  inputWrapper.innerHTML = `
+  <button id="telBtn">Telefon</button>
+  <button id="instagramBtn">Instagram</button>
+  <button id="facebookBtn">Facebook</button>
+  `
+    document.getElementById('telBtn').addEventListener('click', handleTelInput);
+    document.getElementById('instagramBtn').addEventListener('click', handleInstagramInput);
+    document.getElementById('facebookBtn').addEventListener('click', handleFacebookInput);
+}
+
+const handleTelInput = () => {
+  showMessage("Telefon", 'user')
+
+  setTimeout(() => showTelefonMessage(), 1800)
+ }
+
+const handleInstagramInput = () => {
+  showMessage("Instagram", 'user')
+
+  setTimeout(() => showInstagramMessage(), 1800)
+}
+
+const handleFacebookInput = () => {
+  showMessage("Facebook", 'user')
+
+  setTimeout(() => showFacebookMessage(), 1800)
+}
+
+const showTelefonMessage = () => {
+  showMessage("Das ist meine Telefonnummer, bitte ruf mich einfach an oder schreib mir eine Nachricht", 'bot')
+
+  setTimeout(() => showByeMessage(), 2000)
+}
+
+const showInstagramMessage = () => {
+  showMessage("Das ist der Link zu meiner Instagramseite www.instagram/nadelöhr.ch, bitte schreib mir doch eine kurze Nachricht", 'bot')
+
+  setTimeout(() => showByeMessage(), 2000)
+}
+
+const showFacebookMessage = () => {
+  showMessage("Das ist der Link zu meiner Facebookseite www.facebook/nadelöhr.ch, bitte schreib mir doch eine kurze Nachricht", 'bot')
+
+  setTimeout(() => showByeMessage(), 2000)
+}
```

Comment:
Perhaps these could be in one function and the message could be conditionally printed, something like:
```
const showContactMessage = contactType => {
  if (contactType === "instagram") {
    showMessage("Das ist der Link zu meiner Instagramseite www.instagram/nadelöhr.ch, bitte schreib mir doch eine kurze Nachricht", 'bot')
  } else if ...
}
```